kibbeh/src/stories/UserWideButton.story.tsx

```diff
@@ -0,0 +1,25 @@
+import { Story } from "@storybook/react";
+import React from "react";
+import { sampleBaseUser } from "./data/BaseUser";
+import { UserWideButton, UserWideButtonInfoProps } from "../ui/UserWideButton";
+
+export default {
+  title: "UserWideButton",
```

Comment:
Please recheck lines indentations in this file. You use 2 spaces and 4 spaces in different places.